Splash/.gitignore

```diff
@@ -0,0 +1 @@
+SpotifyAPIManager.m
```

Comment:
Is this intended. This would mean any changes to SpotifyAPIManager would not be included in the commit. 
Example for gitignore file: https://github.com/github/gitignore/blob/main/Objective-C.gitignore